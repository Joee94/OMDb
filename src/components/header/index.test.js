import Header from './index';

describe('header', () => {
	describe('when not loading', () => {
		const state = { loading: false };
		const actions = {};

		let returnValue;
		beforeEach(() => {
			returnValue = Header({ state, actions });
		});
		it('should create the header component', () => {
			expect(returnValue).toMatchSnapshot();
		});
	});
	describe('when loading', () => {
		const state = { loading: true };
		const actions = {};

		let returnValue;
		beforeEach(() => {
			returnValue = Header({ state, actions });
		});
		it('should create the header component', () => {
			expect(returnValue).toMatchSnapshot();
		});
	});
});
