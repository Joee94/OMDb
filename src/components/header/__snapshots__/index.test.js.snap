// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header when loading should create the header component 1`] = `
<header
  className="header"
>
  <Title />
  <SearchBar />
</header>
`;

exports[`header when not loading should create the header component 1`] = `
<header
  className="header"
>
  <Title />
  <SearchBar />
</header>
`;
